recipe/meta.yaml

```diff
@@ -13,32 +13,38 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
We should skip `py<36`, see https://github.com/WoLpH/python-utils/blob/1aa3a08e7c32762fcf2f5572de34b1eda4e46806/setup.py#L19

Reply:
oops, saw the approve checkmark and hit merge before seeing this. I'll correct.